opentech/apply/funds/management/commands/migration_base.py

```diff
@@ -80,6 +80,11 @@ def process(self, id):
         except ApplicationSubmission.DoesNotExist:
             submission = ApplicationSubmission(drupal_id=node['nid'])
 
+        # Disable auto_* on date fields so imported dates are used.
+        for field in submission._meta.local_fields:
+            if field.name == "submit_time":
```

Comment:
This field is not part of the ApplicationSubmission model in submissions.py. I find it only in 0005_applicationsubmission.py?

Reply:
Ably demonstrating why inheritance/mixins is a pain

It comes from AbstractFormSubmission: https://github.com/gasman/wagtail/blob/7034cd131774b8971ff3c7424999a28164480f29/wagtail/contrib/forms/models.py#L46